src/GovUk.Education.ExploreEducationStatistics.Public.Data.Processor.Tests/Functions/ProcessInitialDataSetVersionFunctionTests.cs

```diff
@@ -179,8 +79,8 @@ public async Task DuckDbFileIsDeleted()
 
         private async Task CompleteProcessing(Guid instanceId)
         {
-            var function = GetRequiredService<ProcessCompletionOfNextDataSetVersionFunction>();
-            await function.CompleteNextDataSetVersionImportProcessing(instanceId, CancellationToken.None);
+            var function = GetRequiredService<CompleteInitialDataSetVersionProcessingFunction>();
+            await function.CompleteInitialDataSetVersionProcessing(instanceId, CancellationToken.None);
```

Comment:
The name of the private method that this is within should probably be `CompleteInitialDataSetVersionProcessing` rather than `CompleteProcessing`.

Reply:
Man, my head was a jumble that day.  Yes, updated the test class name thanks, and also updated inner class metho0d name to `CompleteInitialDataSetVersionProcessing` from `CompleteProcessing`.

I think I genuinely have trouble with the names of these functions as there are a lot of similar ones, some are grouped in classes with multiple functions, but most are in their own classes now, some explicitly now say "Orchestration" in their names whereas others don't hint at their function type.  I think that for me I could do with adding some details as to the function type to the naming or the organisation of our functions, and personally I'd potentially suggest just giving each Function its own class.